framework/.npmignore

```diff
@@ -0,0 +1,13 @@
+.eslintignore
+.eslintrc.json
+.gitignore
+.lintstagedrc.json
+.nycrc
+.prettierrc.json
+.prettierignore
+cypress.json
+index.html
+Jenkinsfile*
+
+test/
+logs/
```

Comment:
We already have a `src` directory so why not we simplify the ignore file by

```
* 

!src/**/*
!README.md
```

This way every thing will be ignored except content in `src` directory and all README files.